src/pexprs-generateExample.js

```diff
@@ -8,31 +8,50 @@ var common = require('./common');
 var pexprs = require('./pexprs');
 
 // --------------------------------------------------------------------
+// Helpers
+// --------------------------------------------------------------------
+
+function flatten(listOfLists) {
+  return [].concat.apply([], listOfLists);
```

Comment:
Nit: You can do `Array.prototype.concat.apply` to avoid allocating an extra array that just turns into garbage.